cs/commandstation/TrainDbCdi.hxx

```diff
@@ -53,23 +53,20 @@ static const char MOMENTARY_MAP[] =
     "<relation><property>1</property><value>Momentary</value></relation>";
 
 static const char FNDISPLAY_MAP[] =
-    "<relation><property>0</property><value>Unavailable</value></relation>"
-    "<relation><property>1</property><value>Light</value></relation>"
-    "<relation><property>2</property><value>Beamer</value></relation>"
-    "<relation><property>3</property><value>Bell</value></relation>"
-    "<relation><property>4</property><value>Horn</value></relation>"
-    "<relation><property>5</property><value>Shunting mode</value></relation>"
-    "<relation><property>6</property><value>Pantograph</value></relation>"
-    "<relation><property>7</property><value>Smoke</value></relation>"
-    "<relation><property>8</property><value>Momentum off</value></relation>"
-    "<relation><property>9</property><value>Whistle</value></relation>"
-    "<relation><property>10</property><value>Sound</value></relation>"
-    "<relation><property>11</property><value>F</value></relation>"
-    "<relation><property>12</property><value>Announce</value></relation>"
-    "<relation><property>13</property><value>Engine</value></relation>"
-    "<relation><property>14</property><value>Light1</value></relation>"
-    "<relation><property>15</property><value>Light2</value></relation>"
-    "<relation><property>17</property><value>Uncouple</value></relation>"
+    "<relation><property>0</property><value>Unassigned</value></relation>"
+    "<relation><property>1</property><value>Headlight</value></relation>"
+    "<relation><property>4</property><value>Engine</value></relation>"
+    "<relation><property>6</property><value>Announce</value></relation>"
+    "<relation><property>7</property><value>Shunting Mode</value></relation>"
+    "<relation><property>8</property><value>Momentum</value></relation>"
+    "<relation><property>10</property><value>Smoke</value></relation>"
+    "<relation><property>11</property><value>Pantograph</value></relation>"
+    "<relation><property>12</property><value>Far Light</value></relation>"
+    "<relation><property>13</property><value>Bell</value></relation>"
+    "<relation><property>14</property><value>Horn</value></relation>"
+    "<relation><property>15</property><value>Whistle</value></relation>"
+    "<relation><property>74</property><value>Light</value></relation>"
+    "<relation><property>123</property><value>Uncouple</value></relation>"
```

Comment:
This is not the correct uncouple, the right number is 9. I will fix this myself.